### DISCUSSION TITLE
Running Webpack-dev-server through binary still viable?

### DISCUSSION THREAD

**Discussion by nsunga** [webpack/webpack#16828] [Q&A] [answered]
Back in v3 for webpack-dev-server, running the binary like `node_modules/.bin/webpack-dev-server` was acceptable: https://github.com/webpack/webpack-dev-server/tree/v3.7.1#with-the-cli

Im now using v4 for dev server and this seems to have changed: https://github.com/webpack/webpack-dev-server/tree/v4.0.0#with-the-cli

Ive looked into the change log and it looks like the change isnt breaking: https://github.com/webpack/webpack-dev-server/pull/3165

So i just wanted to know if this is still okay to do? ie. `node node_modules/.bin/webpack-dev-server`

This way, I can still add Command Line args to the node call

**Comment by alexander-akait** [comment, marked as answer]
> Ive looked into the change log and it looks like the change isnt breaking: https://github.com/webpack/webpack-dev-server/pull/3165

Yes, we keet it for backward compatibility, and I don't think we will remove it someday, anyway I recommend to use `node node_modules/.bin/webpack`
